components/Card.js

```diff
@@ -0,0 +1,23 @@
+import GlobalStyle from "../styles";
+import Image from "next/image";
+import styled from "styled-components";
+
+const StyleImage = styled(Image)`
+  border-radius: 15px;
+  margin-top: 1rem;
+  margin-left: -1rem;
+  @media screen and (max-width: 768px) {
+    margin-bottom: 0rem;
+  }
+`;
+
+export default function Card({ id, location, type, image }) {
+  return (
+    <>
```

Comment:
If you are only returning this one element, you do not need <> </> there, this is what i am told all the time...